src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -1769,10 +1769,10 @@ def list_resource_links(cmd, scope=None, filter_string=None):
 # endregion
 
 
-def rest_call(cmd, method, url, headers=None, uri_parameters=None,
+def rest_call(cmd, method, uri, headers=None, uri_parameters=None,
```

Comment:
I would recommend making the `az rest` command `is_preview=True` until the signature stabilizes. One thing